packages/timeline-state-resolver/src/conductor.ts

```diff
@@ -114,6 +114,12 @@ export interface StatReport {
 	timelineSizeOld: number
 	estimatedResolveTime: number
 }
+export interface Datastore {
+	[key: string]: {
+		value: any
+		modified: number
```

Comment:
This could use some documentation, what it is (a timestamp) and how it is used (together with `overwrite`).

Possibly also rename it to be more similar to the `overwrite`-property, since the two are closely related. perhaps "overwriteTimestamp"?